src/test/java/seedu/address/ui/CommandBoxTest.java

```diff
@@ -0,0 +1,141 @@
+package seedu.address.ui;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.jupiter.api.BeforeEach;
```

Comment:
Thank you very much for taking on the task of adding UI testing. It is a very laborious process!